src/Behat/Testwork/Exception/Stringer/PHPUnitExceptionStringer.php

```diff
@@ -62,14 +64,14 @@ public function stringException(Exception $exception, $verbosity)
             // Class names are intentionally fully qualified here to maximise clarity - particularly because a future
             // PHPUnit version may reuse a class name in a different namespace.
 
-            if (class_exists(\PHPUnit\Util\ThrowableToStringMapper::class)) {
+            if (class_exists(ThrowableToStringMapper::class)) {
```

Comment:
It's probably OK to lose the fully-qualified names here - particularly now we know that people won't be able to use just the assertions from future PHPUnit versions anyway?

If so we should just remove the comment above.

Reply:
I had not realised about the comment, agree with you, removed